google/ggrc-core
# GGRC-8471 Assessment Template: Change list of default assignees, verifiers

# Issue description

In the assessment template for

Standard, Regulation, Requirement, Objective, Threat, Contract, Policy, Scope objects
the list of options for Default assignees & Default verifiers is the following:
Principal Assignees
Secondary Assignees
Primary Contacts
Secondary Contacts 
Looks like Principal Assignees and Secondary Assignees are not applicable for these object types.

Remove Principal Assignees and Secondary Assignees options
in Default assignees & Default verifiers drop-downs in Assessment Template.

# Steps to test the changes

1. Open a modal to create new Assessment Template.
2. Select any Default Assessment Type.
3. Verify list of Default Assignees/Verifiers.

# Solution description

Removed Principal Assignees and Secondary Assignees values from constant. 
Update default values according to Template Type. 

# Sanity checklist




# PR Review checklist



###
